pkg/controller/machineset/delete_policy.go

```diff
@@ -67,6 +76,34 @@ func oldestDeletePriority(machine *v1alpha1.Machine) deletePriority {
 	return deletePriority(float64(mustDelete) * (1.0 - math.Exp(-d.Seconds()/secondsPerTenDays)))
 }
 
+
+func customDeletePolicy(machine *v1alpha1.Machine) deletePriority {
```

Comment:
Let's call it `defaultDeletePolicy`.
"Custom" implies that a user can set it.

Reply:
Sure, will change the name to avoid unexpected changes.